Set/Game/GameView.swift

```diff
@@ -66,6 +55,68 @@ struct GameView: View {
 //            }
 //            .padding()
         }
+        .navigationTitle("\(gameViewModel.timerTitle)")
+        .navigationBarTitleDisplayMode(.inline)
+        .navigationBarBackButtonHidden(true)
+        .navigationBarItems(
+            leading: Button {
+                gameViewModel.dealThreeMoreCards()
+            } label: {
+                threeMoreCardsButton
+            }
+            .disabled(gameViewModel.isMoreCardAvailable),
+
+            trailing: Button {
+                gameViewModel.startNewGame()
+            } label: {
+                newGameButton
+            }
+        )
+    }
+
+    private var gameBlock: some View {
```

Comment:
In case variable stores a part of `View` (e.g. subview) it can be moved to separate file to not store all in one file, but having in it in separate variable is also good